nekoyume/Assets/_Scripts/Game/Game.cs

```diff
@@ -133,7 +133,12 @@ private IEnumerator Start()
             }
             else
             {
-                yield return L10nManager.Initialize(languageType).ToYieldInstruction();
+                yield return L10nManager.Initialize(languageType.Value).ToYieldInstruction();
+                
+                languageType.Subscribe(_ =>
+                {
+                    L10nManager.SetLanguage(languageType.Value);
+                }).AddTo(gameObject);
```

Comment:
How is this?

```suggestion
                languageType.Subscribe(L10nManager.SetLanguage).AddTo(gameObject);
```

Reply:
Thank you for suggestion! I'll apply it. 😄